state-sync/storage-service/server/src/tests.rs

```diff
@@ -113,7 +113,7 @@ async fn test_get_storage_server_summary() {
     let highest_epoch = 10;
     let expected_server_summary = StorageServerSummary {
         protocol_metadata: ProtocolMetadata {
-            max_epoch_chunk_size: 1000,
+            max_epoch_chunk_size: 100,
```

Comment:
I'm wondering whats the reason for the chunk size chnage?

Reply:
This is because there's a hard limit of 100 epochs embedded in the Diem DB code 😄